docs_src/microservices/device/remote/Ch-RemoteNonSecure.md

```diff
@@ -0,0 +1,202 @@
+This example can be further extended to run multiple instances of device-usb-camera service in multiple nodes.
+
+## Running of the example
+
+1. Set up the two nodes to be ready for remote deployment. Refer to [USB Service Setup](../services/device-usb-camera/walkthrough/setup.md)
+   for system requirements and dependencies such as Git, Docker, Docker Compose, etc. Additionally Golang needs to be installed
+   in the remote node if the device-usb-camera service will be built and run natively.
+
+1. Next step is to install [EdgeX compose](https://github.com/edgexfoundry/edgex-compose) in the host node which will be used to run all EdgeX core services. So clone the `edgex-compose`
+   repository:
+
+     ```bash
+     git clone https://github.com/edgexfoundry/edgex-compose.git
+     ```
+
+1. Checkout the required version:
+
+      ```bash
+      git checkout {{edgexversion}}
+      ```
+
+1. Update the [docker-compose-no-secty.yml](https://github.com/edgexfoundry/edgex-compose/blob/main/docker-compose-no-secty.yml) file by removing the `host_ip` section of all the EdgeX core services. E.g.
+      ```bash
+      host_ip: 127.0.0.1
+      ```
+   The example line provided above should be removed from the services, the host_ip will be provided while running the usb service.
+   Non-EdgeX core services such as device-rest, device-virtual, app-rules-engine, etc. can be removed or commented out if needed.
+
+1. Run EdgeX core services:
+
+      ```bash
+      make run no-secty
+      ```
+
+   1. Verify the services are up and running:
+
+      ```bash
+      docker ps 
+      ```
+
+   1. Clone the [device-usb-camera](https://github.com/edgexfoundry/device-usb-camera) service repository:
+
+       ```bash
+       git clone https://github.com/edgexfoundry/device-usb-camera.git
+       ```
+
+   1. Checkout the required version:
+
+       ```bash
+       git checkout {{edgexversion}}
+       ```
+
+   1. Follow the guide below to build and run the `device-usb-camera` service in Docker or natively.
+
+=== "Docker"
+
+    1. Build the service from the `main` directory:
+
+         ```bash
+         make docker
+         ```
+
+    1. Create `docker-compose.yml` file from anywhere in the remote node to run the device service in Docker. Copy the content below into the compose file and edit with approriate values:
+
+        ```bash
+        name: edgex
+        services:
+            device-usb-camera:
+            container_name: edgex-device-usb-camera
+            device_cgroup_rules:
+            - c 81:* rw
+            environment:
+              EDGEX_SECURITY_SECRET_STORE: "false"
+              EDGEX_REMOTE_SERVICE_HOSTS: "<host-ip-address>,<remote-ip-address>,<service-bind-address>"
+              #E.g.EDGEX_REMOTE_SERVICE_HOSTS: "172.118.1.92,172.118.1.167,0.0.0.0"
+            hostname: edgex-device-usb-camera
+            image: <device-usb-camera-docker-image previously built>
```

Comment:
In docker there is no reason to build the image locally. Use the published image
```suggestion
            image: <published image for device-usb-camera>
            #E.g.  image: nexus3.edgexfoundry.org:10004/device-usb-camera:latest or
            #        image: edgexfoundry/device-usb-camera:{{edgexversion}}.0
```